docs/docs/workflows/upgrade.md

```diff
@@ -51,3 +51,33 @@ If you are interested, you can monitor pods restarting in the `kube-system` name
 Image and Kubernetes upgrades take longer.
 For each node in your cluster, a new node has to be created and joined.
 The process usually takes up to ten minutes per node.
+
+## Check the status
+
+Upgrades are a asynchronous operation.
+Meaning that after you run `upgrade apply` it will take a while until the upgrade has completed.
+To understand if an upgrade is finished you can run:
```

Comment:
```suggestion
Upgrades are asynchronous operations.
After you run `upgrade apply`, it will take a while until the upgrade has completed.
To understand if an upgrade is finished, you can run:
```